Microsoft/WebTemplateStudio
# Allow one azure hosting service at a time

**Changes**
- Once a hosting service is added, all the other hosting services should be disabled so that only one can be added at a time

**How To Test**
- Adding App Service should disable Azure Functions and vice versa
- Removing a service should enable both cards

Issue #971

![image](https://user-images.githubusercontent.com/21695878/62171719-c346de80-b2e4-11e9-8831-c125cad770dd.png)
